auv_control_demos/README.md

```diff
@@ -0,0 +1,321 @@
+# AUV Control Demos
+
+This package contains examples that demonstrate how to use the controllers in
+`auv_controllers`. There are two examples. The first shows how to launch just a
+single controller on its own. The second example shows how to chain the
+controllers together to a mock hardware interface for an AUV.
+
+## Example 1: Individual Controller
+
+This example shows how to launch an individual auv controller connect them to a
+mock hardware interface. We launch the integral sliding mode controller which
```

Comment:
I don't think you need to mention that this uses a mock hardware interface

Reply:
okay.